src/com/lilithsthrone/game/character/fetishes/Fetish.java

```diff
@@ -412,6 +412,67 @@ public CorruptionLevel getAssociatedCorruptionLevel() {
 		}
 	},
 	
+	FETISH_COCK_ADDICT(60,
+			"cock addict",
+			"others cocks",
+			"fetish_cock_addict",
+			Fetish.BASE_RARE_EXPERIENCE_GAIN,
+			Colour.GENERIC_ARCANE,
+			null,
+			Util.newArrayListOfValues("<span style='color:"+ Colour.GENERIC_GOOD.toWebHexString()+ ";'>Unlocks</span> <span style='color:"+ Colour.GENERIC_SEX.toWebHexString()+ ";'>cock addict tease</span>",
+					"<span style='color:"+ Colour.GENERIC_BAD.toWebHexString()+ ";'>Weak to</span> <span style='color:"+ Colour.GENERIC_SEX.toWebHexString()+ ";'>dick dealer tease</span>"),
+			null) {
+
+		@Override
+		public String getDescription(GameCharacter owner) {
+			if (owner.isPlayer())
+				return "You are hopelessly addicted to cock. You really don't care who's providing it; all you want is for your holes to be full of delicious, meat..."
```

Comment:
what's the comma doing between delicious and meat?

Reply:
it represents some other descriptive elements not suitable for young eyes